README.md

```diff
@@ -1 +1,154 @@
-# traffic-state-estimation
\ No newline at end of file
+# Traffic State Estimation Applications for Eclipse MOSAIC
+
+This repository contains applications for Floating Car Data (FCD) based Traffic State Estimation (TSE) to be used with
+[Eclipse MOSAIC](https://eclipse.dev/mosaic).
+Included metrics are partly based on a paper by Yoon et al.[^1].
+Furthermore, we provide additional MOSAIC configuration files to be used with the 
+[BeST Scenario](https://github.com/mosaic-addons/best-scenario)[^2], to reproduce and validate results and develop your own TSE metrics.
+
+## Documentation
+<p align="center"><img width="50%" src=".github/fcd-system-overview.png"/></p>
+The picture above depicts the fundamental structure and functioning of this application suite.
+The functionality is split across two applications `TseServerApp` and the `FcdTransmitterApp`.
```

Comment:
I not sure why, but somehow the `` ` `` are not recognized here and `TseServerApp` is not formatted correctly

Reply:
A white line after the image seemed to have done the trick.